packages/media/src/utils/types.ts

```diff
@@ -184,6 +184,13 @@ export interface ClientMetadataReceivedEvent {
     };
 }
 
+export interface ClientUnableToJoinEvent {
```

Comment:
the `client_unable_to_join` is a signal event afaik, why do we need it in the `media` utils? 🤔

Reply:
That's where the rest of the event payload types are I think

Reply:
ah but I did forget to add it to the bigger list of signal events 💡